src/aspire/reconstruction/estimator.py

```diff
@@ -35,6 +64,37 @@ def __init__(self, src, basis, batch_size=512, preconditioner="circulant"):
                 f" Given src.L={src.L} != {basis.nres}"
             )
 
+        # Checkpoint configuration
+        if checkpoint_iterations is not None:
+            try:
+                checkpoint_iterations = int(checkpoint_iterations)
+            except ValueError:
+                # Sentinel value to emit a more descriptive message below.
+                checkpoint_iterations = -1
+
+            if not checkpoint_iterations > 0:
+                raise ValueError(
+                    "`checkpoint_iterations` should be a positive integer or `None`."
+                )
+        self.checkpoint_iterations = checkpoint_iterations
+
+        # Create checkpointing dirs as needed
+        parent = Path(checkpoint_prefix).parent
```

Comment:
Would probably be a rare case, but if `checkpoint_prefix` is `None` I think this breaks.

Reply:
Yes, that's a bug, pushing a small fix, thanks.